docs/mkdocs/docs/api/macros/nlohmann_json_serialize_enum _strict.md

```diff
@@ -0,0 +1,93 @@
+# NLOHMANN_JSON_SERIALIZE_ENUM_STRICT
+
+```cpp
+#define NLOHMANN_JSON_SERIALIZE_ENUM_STRICT(type, conversion...)
+```
+
+The `NLOHMANN_JSON_SERIALIZE_ENUM_STRICT` allows to define a user-defined serialization for every enumerator.
```

Comment:
This is missing some words.

> The `...` macro allows defining a ...
or
> The `...` macro allows one to define a ...

I can see this is basically the same text as in the original, it should probably be updated there too.